src/sentry/static/sentry/app/views/organizationIntegrations/installedIntegration.jsx

```diff
@@ -37,13 +39,18 @@ export default class InstalledIntegration extends React.Component {
     );
   }
 
+  mergeIntegration() {}
+
   renderDisableIntegration(integration) {
-    const message = `You must uninstall this integration from
-      %s in order to delete this integration in Sentry.`;
+    let {body, action_text} = integration.provider.aspects.disable_dialog;
+    let message = <Alert type="info" icon="icon-circle-exclamation">{body}</Alert>;
+
     return (
       <Confirm
-        confirmText={t('Remove on GitHub')}
-        message={t(message, integration.provider.key)}
+        confirmText={t(action_text)}
```

Comment:
Can we do the translation in django actually? I think this is consistent with the rest of the metadata stuff? maybe I'm wrong